scripts/test_django.py

```diff
@@ -0,0 +1,702 @@
+            # end of the header and that it's not time to parse the content of the message
+            elif in_header and line == LAST_END_MESSAGE:
+                # Stop parsing header data
+                in_header = False
+            # If we're in a message and we encounter the start of a new message or we're not in
+            # the header reading stage and see the end of a header, we know we've reached the last of the messages in stderr
+            # Lines following will consist of extra data, such as the number of tests run and failed
+            elif in_message and (line == START_MESSAGE or not in_header and line == LAST_END_MESSAGE):
+                current_content = current_content.strip()
+                message = TestMessage(status=current_status, content=current_content, description=current_description)
+                self.messages.append(message)
+                in_message = False
+                current_content = ""
+                current_status = ""
+                current_description = None
+                in_header = False
+            # Otherwise, if we're in a message, we want to append what we're reading to whatever we're currently building
+            elif in_message:
+                # If we're in the header, the status will be the first encountered line, so if we're in a header 
+                # and we don't have a status, assign this line to the status
+                if in_header and not current_status:
+                    current_status = line.strip()
+                # If we're still in the header, we know we want to attach this to the description
+                elif in_header:
+                    # Since we've read a new line, we want to add a line separator to make sure that this data is 
+                    # separated from the previously read
+                    if current_description:
+                        current_description += os.linesep
+
+                    current_description += line
+                # Otherwise, we want to just tack the read data onto the content of the message we're building
+                else:
+                    # If we're on the final line of the messages, just tack on a new line to our content, 
+                    # otherwise attach the new content after a new line in order to keep the natural separation from stderr
+                    current_content += os.linesep if line == LAST_END_MESSAGE else f"{os.linesep}{line}"
+
+        # If it's detected that we're still parsing and have reached the end of the document, we want to attach 
+        # what we have as a new message
+        if current_content and in_message:
+            current_content = current_content.strip()
+            message = TestMessage(status=current_status, content=current_content, description=current_description)
+            self.messages.append(message)
+
+    def print(self, verbose: bool = None, quiet: bool = None):
```

Comment:
I think both `verbose` and `quiet`'s default value should be `False` instead of `None` here.